src/tests/BaGet.Tests/TestServerBuilder.cs

```diff
@@ -24,47 +26,70 @@ public class TestServerBuilder
         /// </summary>
         private TestServerBuilder()
         {
+            Configuration = new Dictionary<string, string>();
         }
 
         private ITestOutputHelper _helper;
         private LogLevel _minimumLevel = LogLevel.None;
 
+        /// <summary>
+        /// In Memory representation of Config Settings
+        /// </summary>
+        public Dictionary<string, string> Configuration { get; private set; }
+
         /// <summary>
         /// Xunit.ITestOutputHelper is used as Logging-Target (Microsoft.Extensions.Logging)
         /// </summary>
         /// <param name="helper"></param>
         /// <param name="minimumLevel"></param>
         /// <returns></returns>
-        public TestServerBuilder TraceToTestOutputHelper(ITestOutputHelper helper, LogLevel minimumLevel )
+        public TestServerBuilder TraceToTestOutputHelper(ITestOutputHelper helper, LogLevel minimumLevel)
         {
             _helper = helper ?? throw new ArgumentNullException(nameof(helper));
             _minimumLevel = minimumLevel;
             return this;
         }
 
         /// <summary>
-        /// Empty means: 
-        ///  - Sqlite Database with a new/empty file
-        ///  - StorageType FileSystem based on a empty/temporary directory
+        /// Not configured Test Server Builder
         /// </summary>
         /// <returns></returns>
-        public static TestServerBuilder Empty()
+        public static TestServerBuilder Create()
         {
             return new TestServerBuilder();
         }
 
-        public TestServer Build()
+        private readonly string DatabaseTypeKey = $"{nameof(BaGetOptions.Database)}:{nameof(DatabaseOptions.Type)}";
+        private readonly string ConnectionStringKey = $"{nameof(BaGetOptions.Database)}:{nameof(DatabaseOptions.ConnectionString)}";
+        private readonly string StorageTypeKey = $"{nameof(BaGetOptions.Storage)}:{nameof(StorageOptions.Type)}";
+        private readonly string MirrorEnableReadThroughCachingKey = $"{nameof(BaGetOptions.Mirror)}:{nameof(MirrorOptions.EnableReadThroughCaching)}";
+        private readonly string FileSystemStoragePathKey = $"{nameof(BaGetOptions.Storage)}:{nameof(FileSystemStorageOptions.Path)}";
+
+        /// <summary>
+        /// Creates a subdirectory: Path.GetTempPath() + Guid  
+        /// and uses this as location for 
+        /// - Sqlite file  => .\BaGet.db 
+        /// - FilePackageStorageService => .\Packages\*.*
+        /// </summary>
+        /// <returns></returns>
+        public TestServerBuilder UseEmptyTempFolder()
```

Comment:
>UseEmptyTempFolder [](start = 33, length = 18)

This is great! :)

Reply:
>If you like this pattern, maybe Build could do this for TraceToTestOutputHelper as well?

good idea, but the problem is how inject the ITestOutputHelper from xUnit into the Builder ?
ok we can move the "Helper" Argument from the method "TraceToTestOutputHelper" to the "Create" method....to you mean that ?